### DISCUSSION TITLE
Cipher list context that support FIPS crypto library

### DISCUSSION THREAD

**Discussion by ramkumarpvs** [awslabs/aws-crt-cpp#507] [Q&A] [answered]
What should be the cipher list  for aws_tls_cipher_pref  that support FIPS mode crypto library like OpenSSL or BSafe? 

Looks like default list 
 AWS_IO_TLS_CIPHER_PREF_SYSTEM_DEFAULT which would be set has FIPS support issues while using Aws::Crt::Io::TlsContextOptions::InitClientWithMtls API.

**Comment by ramkumarpvs** [comment]
@jannostern , @dagnir , @bretambrose , @hyandell 
Do you have any information on the question ?

**Comment by bretambrose** [comment, marked as answer]
As it currently stands there is no FIPS-specific support in the CRT or its supporting libraries.  FIPS suport via s2n/aws-lc is planned but I don't have any timeline.  If you're using a FIPS-compliant openssl (rather than aws-lc) then you may want to ask a similar question on the s2n repository; I certainly do not have enough background to give any correct guidance on that matter.
